runtime/src/main/java/org/corfudb/protocols/wireprotocol/ReadRequest.java

```diff
@@ -16,36 +18,24 @@
 @AllArgsConstructor
 public class ReadRequest implements ICorfuPayload<ReadRequest> {
 
-    final Range<Long> range;
-    @SuppressWarnings({"checkstyle:abbreviationaswordinname", "checkstyle:membername"})
-    final UUID streamID;
+    final Set<Long> set;
```

Comment:
Hm, let's not use set here - use a list. If the caller requests multiple addresses twice, we'll just return it, no need to enforce this.